.github/workflows/deploy.yml

```diff
@@ -98,9 +98,27 @@ jobs:
     - name: create tags
       id: tags
       run: |
+        # TODO(asmacdo) github.repository_owner is wrong. This should be a quay account, `operator-framework` happens to be the same in
+        # gh and quay.
         IMG=quay.io/${{ github.repository_owner }}/${{ matrix.id }}
         echo ::set-output name=tags::$(.github/workflows/get_image_tags.sh "$IMG" "v")
 
+    - name: create ansible 2.9 tag
```

Comment:
TODO: this one can be recombined with #98 since this lives in ansible-operator again.